vendor/github.com/gorilla/rpc/LICENSE

```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2012 Rodrigo Moraes. All rights reserved.
```

Comment:
This seems like a bunch of import for a couple of small methods.  Any way we could just write new functions for encode/decode?